app/assets/stylesheets/to_change_everything/_espanol.scss

```diff
@@ -0,0 +1,528 @@
+.espanol {
+    /*--- Highlight language in mobile language menu ---*/
+    #lang1 > li:nth-child(11) > a:nth-child(1) {
```

Comment:
This seems fragile. All of these will be thrown off when we add new languages to the mix.

What if you change the `<span class="check>` to `<span class="check check-espanol>`?

Then this rule selector could be `.espanol .check-espanol`.

Reply:
fixed